app.py

```diff
@@ -0,0 +1,57 @@
+import time
+import os
+import subprocess
+import logging
+
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+class FluentdManager:
+    def __init__(self):
+        logger.info('Starting...')
+
+    def _parse_additional_fieds(self, additional_fields):
+        try:
+            configuration_start = ['<filter> \n',
+                                   '@type record_modifier\n', '<record>\n']
+            configuration_end = ['</record>\n', '</filter> \n']
+            for field in additional_fields:
+                parsed_field = [ad.strip()
+                                for ad in field.split("=")]
+                if len(parsed_field) > 1:
+                    configuration_start.append(
+                        parsed_field[0] + ' "' + parsed_field[1] + '" \n')
+            full_configuration = configuration_start + configuration_end
+            return ''.join(full_configuration)
+        except:
+            logger.error(
+                "Please check your ADDITIONAL_FIELDS,  pattern to use it should be fieldName1=fieldValue1,fieldName2=fieldValue2")
+
+    def populate_additional_fields(self):
+        additional_fields = [ad.strip()
+                             for ad in os.environ["ADDITIONAL_FIELDS"].split(",")]
+        logger.debug('Length of ADDITIONAL_FIELDS is: ',
+                     len(additional_fields))
+        if len(additional_fields) > 0:
+            extra_configuration = self._parse_additional_fieds(
+                additional_fields)
+            try:
+                with open('./fluentd/etc/fluent_record_modifier.conf', "w") as file:
+                    file.write(extra_configuration)
+            except Exception as e:
+                logger.error(e)
+
+    def run(self):
+        try:
+            subprocess.run(["/bin/entrypoint.sh", "fluentd"])
+        except subprocess.CalledProcessError as e:
```

Comment:
Exception here is too specific, so it doesn't catch other exception. When I ran the docker and then tried to terminate Fluentd with a `^C`, it threw this exception:
```sh
Traceback (most recent call last):
  File "app.py", line 57, in <module>
    w.run()
  File "app.py", line 48, in run
    subprocess.run(["/bin/entrypoint.sh", "fluentd"])
  File "/usr/lib/python3.7/subprocess.py", line 474, in run
    stdout, stderr = process.communicate(input, timeout=timeout)
  File "/usr/lib/python3.7/subprocess.py", line 931, in communicate
    self.wait()
  File "/usr/lib/python3.7/subprocess.py", line 990, in wait
    return self._wait(timeout=timeout)
  File "/usr/lib/python3.7/subprocess.py", line 1624, in _wait
    (pid, sts) = self._try_wait(0)
  File "/usr/lib/python3.7/subprocess.py", line 1582, in _try_wait
    (pid, sts) = os.waitpid(self.pid, wait_flags)
KeyboardInterrupt
```
It shouldn't throw an exception for keyboard interrupt.